package.json

```diff
@@ -12,45 +12,65 @@
     "ionic:serve": "ionic-app-scripts serve"
   },
   "dependencies": {
-    "@angular/animations": "^4.2.5",
-    "@angular/common": "4.2.4",
-    "@angular/compiler": "4.2.4",
-    "@angular/compiler-cli": "4.2.4",
-    "@angular/core": "4.2.4",
-    "@angular/forms": "4.2.4",
-    "@angular/http": "4.2.4",
-    "@angular/platform-browser": "4.2.4",
-    "@angular/platform-browser-dynamic": "4.2.4",
-    "@ionic-native/contacts": "^3.13.1",
-    "@ionic-native/core": "3.12.1",
-    "@ionic-native/splash-screen": "3.12.1",
-    "@ionic-native/status-bar": "3.12.1",
+    "@angular/animations": "^4.1.3",
+    "@angular/common": "4.1.3",
+    "@angular/compiler": "4.1.3",
+    "@angular/compiler-cli": "4.1.3",
+    "@angular/core": "4.1.3",
+    "@angular/forms": "4.1.3",
+    "@angular/http": "4.1.3",
+    "@angular/platform-browser": "4.1.3",
+    "@angular/platform-browser-dynamic": "4.1.3",
+    "@ionic-native/contacts": "4.0.1",
+    "@ionic-native/core": "4.0.1",
+    "@ionic-native/in-app-browser": "4.0.1",
+    "@ionic-native/splash-screen": "4.0.1",
+    "@ionic-native/status-bar": "4.0.1",
     "@ionic/storage": "2.0.1",
     "@ngrx/core": "^1.2.0",
-    "@ngrx/effects": "^2.0.3",
-    "@ngrx/store": "^2.2.2",
+    "@ngrx/effects": "^2.0.4",
+    "@ngrx/store": "^2.2.3",
     "@ngrx/store-devtools": "^3.2.4",
-    "ionic-angular": "3.4.2",
+    "cordova-android": "^6.2.3",
+    "cordova-plugin-compat": "^1.1.0",
+    "cordova-plugin-console": "^1.0.5",
+    "cordova-plugin-contacts": "^2.3.1",
+    "cordova-plugin-device": "^1.1.4",
+    "cordova-plugin-inappbrowser": "^1.7.1",
+    "cordova-plugin-splashscreen": "^4.0.3",
```

Comment:
- `splashscreen` is used to show _welcome/loading_ page just before the app loads. I think it is important.
 - `inappbrowser` "is suppoused to" let you launch other apps or web pages from the app. if it works as expected, we will use it to launch the native history app (call log).
 - `cordova-plugin-device` for device information, it also has the **onDeviceReady** event which we use.

Currently we are using all of them.